train_gpt2.cu

```diff
@@ -425,7 +426,7 @@ void encoder_forward(float* out,
 void layernorm_forward(float* out, float* mean, float* rstd,
                        float* inp, float* weight, float* bias,
                        int B, int T, int C) {
-    const int block_size = 1024;
+    const int block_size = 256;
```

Comment:
You changed the block_size on layernorm

Reply:
I was previously running into resource unavailable error due to limited registers. To address this, I had reduced the block_size to 256.  However, I do not see this error in the current top of the tree. I am going to close this PR and raise a new PR that resolves only the warnings.